code/modules/clothing/suits/labcoat.dm

```diff
@@ -12,67 +12,67 @@
 		"Vox" = 'icons/mob/species/vox/suit.dmi'
 		)
 
-	verb/toggle()
-		set name = "Toggle Labcoat Buttons"
-		set category = "Object"
-		set src in usr
+/obj/item/clothing/suit/storage/labcoat/verb/toggle()
+	set name = "Toggle Labcoat Buttons"
+	set category = "Object"
+	set src in usr
 
-		if(!usr.canmove || usr.stat || usr.restrained())
-			return 0
+	if(!usr.canmove || usr.stat || usr.restrained())
+		return 0
 
-		switch(icon_state)
-			if("labcoat_open")
-				src.icon_state = "labcoat"
-				usr << "You button up the labcoat."
-			if("labcoat")
-				src.icon_state = "labcoat_open"
-				usr << "You unbutton the labcoat."
-			if("labcoat_cmo_open")
-				src.icon_state = "labcoat_cmo"
-				usr << "You button up the labcoat."
-			if("labcoat_cmo")
-				src.icon_state = "labcoat_cmo_open"
-				usr << "You unbutton the labcoat."
-			if("labcoat_gen_open")
-				src.icon_state = "labcoat_gen"
-				usr << "You button up the labcoat."
-			if("labcoat_gen")
-				src.icon_state = "labcoat_gen_open"
-				usr << "You unbutton the labcoat."
-			if("labcoat_chem_open")
-				src.icon_state = "labcoat_chem"
-				usr << "You button up the labcoat."
-			if("labcoat_chem")
-				src.icon_state = "labcoat_chem_open"
-				usr << "You unbutton the labcoat."
-			if("labcoat_vir_open")
-				src.icon_state = "labcoat_vir"
-				usr << "You button up the labcoat."
-			if("labcoat_vir")
-				src.icon_state = "labcoat_vir_open"
-				usr << "You unbutton the labcoat."
-			if("labcoat_tox_open")
-				src.icon_state = "labcoat_tox"
-				usr << "You button up the labcoat."
-			if("labcoat_tox")
-				src.icon_state = "labcoat_tox_open"
-				usr << "You unbutton the labcoat."
-			if("labgreen_open")
-				src.icon_state = "labgreen"
-				usr << "You button up the labcoat."
-			if("labgreen")
-				src.icon_state = "labgreen_open"
-				usr << "You unbutton the labcoat."
-			if("labcoat_mort_open")
-				src.icon_state = "labcoat_mort"
-				usr << "You button up the labcoat."
-			if("labcoat_mort")
-				src.icon_state = "labcoat_mort_open"
-				usr << "You unbutton the labcoat."
-			else
-				usr << "You attempt to button-up the velcro on your [src], before promptly realising how retarded you are."
-				return
-		usr.update_inv_wear_suit()	//so our overlays update
+	switch(icon_state)
+		if("labcoat_open")

```

Comment:
I think it was an 'if it ain't broke' mentality that got them to hang onto it heh